### PR TITLE
v0.180.0

### PR DESCRIPTION
**Pull request by mctofu**
diff: https://github.com/dependabot/dependabot-core/compare/v0.179.0...v0.180.0-release-notes

## v0.180.0, 17 March 2022

- Bump to `go` `1.18` (@jeffwidman) [#4853](https://github.com/dependabot/dependabot-core/pull/4853)
- Fix Bundler 1 spec failures (@deivid-rodriguez) [#4869](https://github.com/dependabot/dependabot-core/pull/4869)
- Require Ruby 2.7 (@deivid-rodriguez) [#4864](https://github.com/dependabot/dependabot-core/pull/4864)
- Bugfix: Include space in regex capturing group (@jeffwidman) [#4868](https://github.com/dependabot/dependabot-core/pull/4868)
- Get bundler specs green when run on Ruby 3.1 (@deivid-rodriguez) [#4841](https://github.com/dependabot/dependabot-core/pull/4841)

### CODE REVIEW

**Comment by ulgens** [comment]
@mctofu Is there any change including https://github.com/dependabot/dependabot-core/pull/4874 in this version?

**Comment by jeffwidman** [comment]
@ulgens from the outside, it seems the core team releases multiple times a week... for example `0.179.0` was released yesterday. So IMO I wouldn't worry about trying to catch a particular release. Once your PR is merged, it'll likely be released/deployed within a few days at most.

**Comment by ulgens** [comment]
@jeffwidman Got it, thank you.

File: CHANGELOG.md
Comment (jakecoffman) [inline CHANGELOG.md:8]: Missed one?

```suggestion
- NuGet version comparison fix [#4862](https://github.com/dependabot/dependabot-core/pull/4862)
```

**Review by jakecoffman — APPROVED** [review]

**Comment by jurre** [comment]
I've merged a few PR's since this was opened, so I'm going to hijack this PR and add the new changes in 😄

File: CHANGELOG.md
Comment (jurre) [reply CHANGELOG.md:11]: This changelog is autogenerated from the merge commits since the last release, it's weird that it missed this 😕

Comment (jakecoffman) [reply CHANGELOG.md:11]: It's probably because I squashed and it looks for merge commits. I'll stick to merge commits so the script keeps working!

Comment (jurre) [reply CHANGELOG.md:12]: Yep, that'd explain it!

**Pull request merged**
